integration-cli/docker_api_logs_test.go

```diff
@@ -82,3 +86,96 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
```

Comment:
Okay, I'll try to give this a go this week, but I don't have much time. Is this auto-termination feature a dealbreaker for release? Just trying to get an idea of what's needed before merging, since this has been in review for around 3 months.